rootfs/rootfs/usr/local/etc/init.d/docker

```diff
@@ -5,78 +5,38 @@
 #import settings from profile (e.g. HTTP_PROXY, HTTPS_PROXY)
 test -f '/var/lib/boot2docker/profile' && . '/var/lib/boot2docker/profile'
 
-: ${DOCKER_HOST:='-H tcp://0.0.0.0:2375'}
-: ${DOCKER_TLS:=auto}
 : ${DOCKER_STORAGE:=auto}
 : ${DOCKER_DIR:=/var/lib/docker}
 : ${DOCKER_ULIMITS:=1048576}
 : ${DOCKER_LOGFILE:=/var/lib/boot2docker/docker.log}
 
-: ${CERTDIR:=/var/lib/boot2docker/tls/}
```

Comment:
Is there any reason to retain this directory (or its contents) in the `boot2docker` image?